content/en/sensitive_data_scanner/guide/redact_all_emails_except_from_specific_domain_logs.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Redact All Emails Except from a Specific Domain in Logs
+disable_toc: false
+further_reading:
+- link: "/sensitive_data_scanner/"
+  tag: "Documentation"
+  text: "Set up Sensitive Data Scanner"
+---
+
+## Overview
+
+This guide walks you through how to redact all emails, except the ones from a specific email domain (for example, `@test.com`), in your logs.
+
+## Set up a grok processor
+
+If the email domain you do not want redacted is not an existing log attribute, set up a grok processor to identify all logs with the email domain and add it as an attribute.
+
+1. Navigate to [Log Pipeline][1].
+1. Select your pipeline.
+1. Click **Add processor**.
+1. Select **Grok Parser**.
+1. Enter a name for the grok parser.
+1. Define the parsing rules to identify all logs with the email address. For example, if these are the log messages that contain email addresses with the domain:
+    ```
+    message successfully sent to 123@test.com
+    ```
+    ```
+    message successfully received from 256@test.com
+    ```
+    Then use the following parsing rules:
+    ```
+    MyParsingRule1 message successfully sent to %{notSpace:user_handle}@%{notSpace:domain}
+
+    MyParsingRule2 message successfully received from %{notSpace:user_handle}@%{notSpace:domain}
+    ```
+    **Note:** You don't need to keep the username. For example, if you want to redact all emails with the domain `test.com`, then for an email like `hello@test.com`, discard the username `hello` and just keep the domain `test.com`.
+1. Click **Save**.
+
+Navigate to [Log Explorer][2] to confirm that new logs coming in with those emails are getting processed as expected.
+
+{{< img src="sensitive_data_scanner/guides/domain_attribute.png" alt="The domain attribute in log side panel" style="width:80%;" >}}
+
+## Add the email domain attribute as a facet
+
+1. In [Log Explorer][2], select a log that contains the email with the specified domain.
+1. Click on the cog next to the domain attribute you just created.
+1. Select **Create facet for...**.
+1. Optionally, add it to a group in the **Advanced Options** section.
+1. Click **Add**.
+
+## Configure the Sensitive Data Scanner scanning group to filter out logs with the domain attribute
+
+Update your Sensitive Data Scanner's scanning group to filter out logs with the domain attribute, so only logs that do not have that email domain are redacted.
+
+1. Navigate to the Sensitive Data Scanner [Configuration][3] page.
+1. Click the pencil icon to the left of the scanning group you want to update.
+1. In the **Filter** field, add the domain attribute so those logs are filtered out. For example, to filter out logs with the email domain `test.com`, add `-@domain:test.com` to the filter query.
```

Comment:
```suggestion
1. In the **Filter** field, add the domain attribute so that logs with that attribute are filtered out. For example, to filter out logs with the email domain `test.com`, add `-@domain:test.com` to the filter query.
```